.github/workflows/build.yml

```diff
@@ -1,6 +1,35 @@
 name: "Build"
 on: [push, pull_request]
 jobs:
+  linux-clang-ASanAndUBSan-build:
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        clang: [8]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+            # Nix Flakes doesn't work on shallow clones
+            fetch-depth: 0
+      - uses: cachix/install-nix-action@v12
+        with:
+          install_url: https://github.com/numtide/nix-flakes-installer/releases/download/nix-2.4pre20201221_9fab14a/install
+          extra_nix_config: |
+            experimental-features = nix-command flakes ca-references
+      - name: nix build hobbesPackages/clang-${{ matrix.clang }}-ASanAndUBSan/hobbes
+        run: |
+          nix build .#hobbesPackages/clang-${{ matrix.clang }}-ASanAndUBSan/hobbes
+      - name: nix log hobbesPackages/clang-${{ matrix.clang }}-ASanAndUBSan/hobbes
+        if: ${{ always() }}
+        run: |
+          nix log  .#hobbesPackages/clang-${{ matrix.clang }}-ASanAndUBSan/hobbes &> ${{ matrix.os }}-clang-${{ matrix.clang }}--ASanAndUBSan-hobbes.log
```

Comment:
Typo? you have 2 dashes in the output filepath `--`, but this is different at line 32 below.

Reply:
Yeah, that's a typo